Header header

GridDSTMetaData metadata

string[] universe

GridDSTMass[] masses